### PR TITLE
adds section on modifying door samples with $_

### PR DESCRIPTION
**Pull request by syzygyzer**
And some minor edits.

### CODE REVIEW

File: tutorials/hoon/doors.md [hunk 1]
```diff
@@ -269,7 +269,7 @@ Gates are useful for defining functions.  But there are many-armed doors as well
 
 Let's write an example door in order to illustrate its features.  Each of the arms in the door will define a simple gate.  Let's bind the door to `c` as we did with the last core.  To make a door we use the `|_` rune:
 
-```hoon
```
Comment (rmariani) [inline tutorials/hoon/doors.md]: any reason the `hoon` tag is removed?

Comment (syzygyzer) [reply tutorials/hoon/doors.md]: its dojo, not hoon

Comment (rmariani) [reply tutorials/hoon/doors.md]: Oh, of course

File: tutorials/hoon/doors.md [hunk 2]
```diff
@@ -394,9 +394,21 @@ Readers with some mathematical background may notice that `~( )` expressions all
 
 Thus, you may think of the `c` door as a function for making functions.  Use the `~(arm c arg)` syntax -- `arm` defines which kind of gate is produced (i.e., which arm of the door is used to create the gate), and `arg` defines the value of `b` in that gate, which in turn affects the product value of the gate produced.
 
+#### Creating Doors with a Modified Sample
+
+In the above example we created a door `c` with sample `b=@` and found that the initial value of `b` was `0`, the bunt value of `@`. We then created new door from `c` by modifying the value of `b`. But we if we wish to define a door with a chosen sample value directly? We make use of the `$_` rune, whose irregular form is simply `_`. To create the door `c` with the sample `b=@` set to have the value `7` in the dojo, we would write
```
Comment (rmariani) [inline tutorials/hoon/doors.md]: Grammar error: `But we if we wish to define a door` (should be "what")

**Pull request merged**